src/test/java/gumtree/spoon/AstComparatorTest.java

```diff
@@ -1541,7 +1541,7 @@ public void test_issue201812() throws Exception {
 		CtClass c2 = Launcher.parseClass("class BehaviorCall implements Call {\n"
 				+ "final AtomicReference failureRef = new AtomicReference<>();\n"
 				+ "final CountDownLatch latch = new CountDownLatch(1);\n" + "enqueue(new Callback() {\n"
-				+ "@override public void onResponse(Call call, Response response) {\n" + "responseRef.set(response);\n"
+				+ "@Override public void onResponse(Call call, Response<T> response) {\n" + "responseRef.set(response);\n"
```

Comment:
I had one question here. If we compare `c1` and `c2`, there is also a difference in the case of `Override` annotation. However, this wasn't appearing as a `diff` before so is this a bug in spoon that it ignored the case of the annotation? Anyway, I still changed the spelling because we want to have correct annotations in the test.

Reply:
it may be a bug. Are the annotations in the ITree?

Reply:
@monperrus There is a node for annotations in `ITree` but it has no label so probably that's why the `diff` is not getting detected.